lms/static/js/learner_dashboard/views/collection_list_view.js

```diff
@@ -0,0 +1,24 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone'],
+         function(
+             Backbone
+         ) {
+            return Backbone.View.extend({
+                initialize: function(data) {
+                    this.childView = data.childView;
+                },
+
+                render: function() {
+                    var cardList = [];
```

Comment:
Nitpick: this variable should be called something like ```childList``` because the view is no longer specific to cards.